core/modules/layout/layout.admin.inc

```diff
@@ -2449,27 +2478,123 @@ function layout_delete_form($form, &$form_state, $layout) {
+ *   The path of the layout (that was just deleted or disabled).
+ * @param string $op
+ *    One of 'delete' or 'disable'.
+ */
+function layout_warn_if_site_info_path($path, $op) {
+  $site_config = config('system.core');
+  switch ($op) {
+    case 'delete':
+      if ($path == $site_config->get('site_frontpage')) {
+        $url = url('admin/config/system/site-information') . '#edit-front-page';
+        backdrop_set_message(t('You have deleted the page at path %path that is used as the home page of the site. You must change the home page on the <a href="!url">site information configuration page</a> to a different path.', array('%path' => $path, '!url' => $url)), 'warning');
+      }
+      if ($path == $site_config->get('site_403')) {
+        $url = url('admin/config/system/site-information') . '#edit-error-page';
+        backdrop_set_message(t('You have deleted the page at path %path that is used as the default 403 (access denied) page of the site. You must change the 403 page on the <a href="!url">site information configuration page</a> to a different path.', array('%path' => $path, '!url' => $url)), 'warning');
+      }
+      if ($path == $site_config->get('site_404')) {
+        $url = url('admin/config/system/site-information') . '#edit-error-page';
+        backdrop_set_message(t('You have deleted the page at path %path that is used as the default 404 (not found) page of the site. You must change the 404 page on the <a href="!url">site information configuration page</a> to a different path.', array('%path' => $path, '!url' => $url)), 'warning');
+      }
+      break;
+
+    case 'disable':
+        if ($path == $site_config->get('site_frontpage')) {
+          $url = url('admin/config/system/site-information') . '#edit-front-page';
+          backdrop_set_message(t('You have disabled the page at path %path that is used as the home page of the site. You must change the home page on the <a href="!url">site information configuration page</a> to a different path.', array('%path' => $path, '!url' => $url)), 'warning');
+        }
+        if ($path == $site_config->get('site_403')) {
+          $url = url('admin/config/system/site-information') . '#edit-error-page';
+          backdrop_set_message(t('You have disabled the page at path %path that is used as the default 403 (access denied) page of the site. You must change the 403 page on the <a href="!url">site information configuration page</a> to a different path.', array('%path' => $path, '!url' => $url)), 'warning');
+        }
+        if ($path == $site_config->get('site_404')) {
+          $url = url('admin/config/system/site-information') . '#edit-error-page';
+          backdrop_set_message(t('You have disabled the the page at path %path that is used as the default 404 (not found) page of the site. You must change the 404 page on the <a href="!url">site information configuration page</a> to a different path.', array('%path' => $path, '!url' => $url)), 'warning');
```

Comment:
`You have disabled the page at %path that was used as the default 404 (not found) page for this site. You should also change the 404 page setting under <a href="!url">site information</a>.`

Reply:
Changed.